Carbon.MassTransit/IServiceCollectionConfiguratorExtensions.cs

```diff
@@ -132,15 +138,64 @@ MassTransitNS.AspNetCoreIntegration.MassTransitHostedService HostedServiceFactor
 
                 var busSettings = massTransitSettings.RabbitMq;
 
-                serviceCollection.UsingRabbitMq((cfg, x) =>
-                {
-                    busFactory(configurator, busSettings, cfg, x);
-                });
+                serviceCollection.UsingRabbitMq((cfg, x) => rabbitMqBusFactory(configurator, busSettings, cfg, x));
```

Comment:
Now, should we only call overloaded methods to avoid code duplication?

Reply:
code duplication removed, thanks for the suggestion :)